app/controllers/users_controller.rb

```diff
@@ -113,6 +113,16 @@ def extlogout
     render :extlogout, :layout => 'login'
   end
 
+  def test_mail
+    user = find_resource(:edit_users)
+    begin
+      MailNotification[:tester].deliver(:user => user)
+    rescue Net::SMTPAuthenticationError, Net::SMTPServerBusy, Net::SMTPSyntaxError, Net::SMTPFatalError, Net::SMTPUnknownError, Timeout::Error => e
+      render :json => e.message, :status => :internal_server_error and return
+    end
+    render :json => {:message => _("Email was sent successfully")}, :status => :ok
+  end
```

Comment:
This means that we use e-mail that is stored in DB, but what if user modifies user's email and then switching tabs before the change actually reach the DB? Then he/she is sending on the old e-mail. Can you pickup the most recent e-mail from the HTML form and pass it in the controller as a param?